src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberMethodSymbol.vb

```diff
@@ -697,17 +697,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return Nothing
             End If
 
-            typeBinder.ReportUseSiteError(diagBag, singleHandleClause.EventMember, eventContainingType)
```

Comment:
`CreateUserDefinedConversion` asserts `convKind.Value IsNot Nothing` and dereferences `convKind.Value` in several places before calling `ReportUseSiteError`.